app/src/main/kotlin/com/x8bit/bitwarden/data/vault/repository/util/InitUserCryptoMethodExtensions.kt

```diff
@@ -0,0 +1,18 @@
+package com.x8bit.bitwarden.data.vault.repository.util
+
+import com.bitwarden.core.InitUserCryptoMethod
+
+/**
+ * Returns the label for the given [InitUserCryptoMethod].
+ */
+fun InitUserCryptoMethod.methodName(): String =
```

Comment:
Since this is not translated, what do you think about converting this to a property and name it `logTag`, or something similar, so it's clear this should only be used for logging? Would also be good to mention that in the KDocs.